### DISCUSSION TITLE
Flatness of a Molecule

### DISCUSSION THREAD

**Discussion by amitlevy** [rdkit/rdkit#4641] [Q&A]
Is there a way to calculate a descriptor that in some way measures the "flatness" of a molecule? Or maybe some combination of descriptors that together would encapsulate that information? I looked at the documentation for different descriptors, and the one thing that seemed most similar to what I wanted was "GetMoleculeBoundsMatrix", but looking at the output I got I think I probably misunderstood what that means (I expected 6 numbers, a minimum and a maximum on each axis, but instead got a 20x20 matrix).

**Comment by jasondbiggs** [comment]
The [plane of best fit](https://www.rdkit.org/docs/source/rdkit.Chem.rdMolDescriptors.html#rdkit.Chem.rdMolDescriptors.CalcPBF) may be what you need.
